interface.go

```diff
@@ -198,7 +198,7 @@ type ESDTRoleHandler interface {
 
 // PayableHandler provides IsPayable function which returns if an account is payable or not
 type PayableHandler interface {
-	IsPayable(address []byte) (bool, error)
+	IsPayable(sndAddress, rcvAddress []byte) (bool, error)
```

Comment:
Not a big fan of having consecutive arguments of the same type as someone can easily switch between them.
Using arguments here, in the interface definition will clutter things up. Can remain as it is.

Reply:
Yes. Will leave as it is.